plugins/coverbrowser.koplugin/listmenu.lua

```diff
@@ -447,10 +465,12 @@ function ListMenuItem:update()
 
             local fontname_title = "cfont"
             local fontname_authors = "cfont"
-            local fontsize_title = 20
-            local fontsize_authors = 18
+            local fontsize_title = 24
+            local fontsize_authors = 22
```

Comment:
I don't want my title and authors bigger when I'll set the same nb of list items I have.
You must have increased that because you need to start from a bigger font size when you have big items (small number of items) - but it shouldn't affect smaller items (large number of items).
Again, it should be a factor/computation vs the available height (possibly keeping the factor the current static nb of items has, to not surprise users with too much change).

Reply:
In the default settings, the title and author pretty much fill the height, so not much change. If there is a lot of whitespace above and below, why not make the text a bit biger?
The loop is not my code... But I'll give the computation a try, but it's probably a lot harder than with the file info, since the title can span more than one line. I shoud be able to decrease the font size based on widget size vs text size rather than by 1, to reduce the number of loops.

Reply:
Whitespace is not a bad thing. It helps direct the eyes.

Reply:
To put it another way, it's a compromise. I think we can all agree that we want the whitespace as small as reasonably possible in such an information dense environment. Based on some of the screenshots below I'm inclined to agree the "after" situation ends up a touch too busy. I think it's because the size seems to be both bigger (reducing needed whitespace) and kind of random?

Reply:
> why not make the text a bit biger?

Well, that's a question of taste :) I personally don't like it, 

> The loop is not my code... 

Indeed. But the starting font size is yours :) May be just finding the right one will make things fine.
Similar to my comments about the right stuff font size, title font size starts at 20, authors' at 18.
So, starting from `item_height * 20 / 64 ` and `item_height * 18/64` should give a similar result as before.
When it does, and only then, we can discuss about these factors (I'll fight against increasing them, but if people agree with your preference, I'll just have a simple factor to change in my own patch.)

Reply:
I like the idea of removing the fixed upper font size in favor of a variable size. With that, there's no real value for me to fight you on the factors :)
I'll try and keep the font size on the default setting (on the emulator) as close to the original visual as possible.

Reply:
(Note that your item_height will be pixels, so already "scaledBySize'd" - so you'd have to invert-scaleBySize to get the font size (as it will be scaleBySize'd by Font:new)

Reply:
scaledBySize reacts to changes in height/width ration much more than changes in height alone. So, using a 64/20 ration for title font:
* font size grows bigger in square viewers _or_
* font size grows smaller in 4/3 screens

depending on whether scaleBySize is used on the font or not... Still wondering about that one. Currently I'm going for same density on 720/540.

Reply:
Ah, I'll have to revert dbg.lua and change all my logs from warn to debug. Sorry 'bout that.

Reply:
Well, I don't know of any squared device :) But I see what you mean. I guess this would also apply when switching to landscape (FileBrowser don't do landscape, but History does if you wanna test).

I'm really fine with your `font_size = math.ceil(nominal * dimen.h / 64)` rule, as I'm then sure we get the ratio we had before on all size.
And testing a bunch of settings, they ALL look really fine and coherent to me now :) so :+1: !

(May be add a comment about why that 64, like "previously, with a fixed size, blah blah was using 64 to get 10 items on gloHD - so let's start from that rule to keep the old aspect")

Another option would be for you to add a `scaleByHeight()`, either here or alongside `fb:scaleBySize(px)` in `base/ffi/framebuffer.lua`.

Reply:
Font sizes are now almost identical with original - one px off in some cases, same size in most. Without explicitly setting `files_per_page`, most users should not see any difference.